interface/patient_file/encounter/find_code_dynamic.php

```diff
@@ -49,6 +50,18 @@
 
 <?php Header::setupHeader(['opener', 'datatables', 'datatables-dt', 'datatables-bs', 'datatables-colreorder']); ?>
 
```

Comment:
Hi  @bradymiller and @tywrenn 

Why not remove the `datatables-dt`, Are any library or feature depend on it? I found it will cause the `datatables-bs` looks odd. e.g. double arrow issue.

Reply:
hi @stu01509 ,
If removing it causes no problems (and avoids the override), then would remove it (I do recall in the past that doing this messed up some of the styling in paging stuff, but maybe it was just a collision of some other css).

Reply:
If possible, would also remove where it's used in other places (then we can remove it completed from package.json).

Reply:
Hi @bradymiller, sir

Thanks for your explanation, I remove ` datatable-dt ` and override CSS, and do a little bit test. It looks like good.

Reply:
@stu01509 , sounds great. once travis gives the ok, I'll bring it in. thanks! -brady